db/database.go

```diff
@@ -660,7 +661,7 @@ func (context *DatabaseContext) SetOnChangeCallback(callback DocChangedFunc) {
 	context.mutationListener.OnDocChanged = callback
 }
 
-func (dbCtx *DatabaseContext) GetServerUUID() string {
+func (dbCtx *DatabaseContext) GetServerUUID(ctx context.Context) string {
```

Comment:
Do we need to do `ctx = context.AddDatabaseLogContext(ctx)` here?  I'm not clear on why that's needed in Close and NewDatabaseContext but wouldn't be needed for other public methods on *DatabaseContext.

Reply:
If it varies by usage, how does a developer adding a new method to *DatabaseContext know whether they should be adding the database log context or not?